ui/src/pages/employees/add.tsx

```diff
@@ -1,41 +1,64 @@
+import { unwrapResult } from '@reduxjs/toolkit';
 import classNames from 'classnames/bind';
-import { Formik, FormikProps } from 'formik';
+import { Formik, FormikHelpers, FormikProps } from 'formik';
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 import * as Yup from 'yup';
 
 import ImageUploadField from 'src/components/Form/ImageUploadField';
 import SelectField from 'src/components/Form/SelectField';
 import TextField from 'src/components/Form/TextField';
 import Layout from 'src/containers/Layout';
 import Header from 'src/containers/Layout/Header';
+import { RootState } from 'src/duck';
+import { DesignationState, fetchDesignation } from 'src/duck/designation';
+import { createEmployee } from 'src/duck/employee';
+import { fetchOrganization, OrganizationState } from 'src/duck/organization';
+
 import style from './style.module.scss';
 
 const c = classNames.bind(style);
 
-const designation = [
-  { id: '1', name: 'Sd', avtar: '' },
-  { id: '2', name: 'De', avtar: '' },
-  { id: '3', name: 'Bd', avtar: '' },
-];
+const AddEmployee: React.FC<FormikProps<FormValues>> = (p) => {
+  const dispatch = useDispatch();
+  const { organizations, isFetching } = useSelector<RootState, OrganizationState>(
+    (state) => state.organization,
+  );
+
+  const { designation, isLoading } = useSelector<RootState, DesignationState>(
```

Comment:
Let's rename similarly named variables and give more context specific names. e.g `isFetching` can become `isFetchingOrganizations`, and `isLoading` can become `isFetchingDesignations`.